manim/mobject/coordinate_systems.py

```diff
@@ -697,6 +700,55 @@ def log_func(x):
         graph.underlying_function = function
         return graph
 
+    def get_implicit_graph(
+        self,
+        func: Callable,
+        color: Color = WHITE,
+        samples: int = 100,
+    ) -> SVGMobject:
```

Comment:
Just  a brief comment, but the code to create the mob should be ported into its own separate class (in `functions.py`) and this method should just be a convenient shorthand for that mob like how `get_graph` currently works. This would let us write methods and work with implicit curves independent of `Axes`.